thorlcr/activities/merge/thmergeslave.cpp

```diff
@@ -270,6 +271,16 @@ class GlobalMergeSlaveActivity : public CSlaveActivity
     void init(MemoryBuffer &data, MemoryBuffer &slaveData) override
     {
         masterMpTag = container.queryJobChannel().deserializeMPTag(data);
+        rwFlags = DEFAULT_RWFLAGS;
+        if (getOptBool(THOROPT_COMPRESS_SPILLS, true))
+        {
+            StringBuffer compType;
+            getOpt(THOROPT_COMPRESS_SPILL_TYPE, compType);
+            unsigned spillCompInfo;
+            setCompFlag(compType, spillCompInfo);
+            if (spillCompInfo)
+                rwFlags |= rw_compress;
```

Comment:
rwFlags |= spillCompInfo;

for consistency with elsewhere?  Really this should be a single piece of common code.... but that can be a separate refactoring.  The same logic seems to be done two different ways.

Reply:
Yes, I meant to have it like that, now fixed.
Have opened separate JIRA to common code like this: https://hpccsystems.atlassian.net/browse/HPCC-32881